webapp/channels/src/components/admin_console/custom_profile_attributes/custom_profile_attributes.tsx

```diff
@@ -112,7 +112,7 @@ const CustomProfileAttributes: React.FC<Props> = (props: Props): JSX.Element | n
 
         props.registerSaveAction(handleSave);
         return () => props.unRegisterSaveAction(handleSave);
-    }, [props.registerSaveAction, props.unRegisterSaveAction, attributes, originalAttributes, attributeKey, props]);
+    }, [props.registerSaveAction, props.unRegisterSaveAction, attributes, originalAttributes, attributeKey]);
```

Comment:
putting the whole props in the dep array was causing infinite rerenders

Reply:
That's a definite downside of how this ESLint warning works since it doesn't really explain itself well. If you want to fix the warning properly, you have to destructure the props used inside of this instead of using `props.foo` inside of the `useEffect`